proto/babylon/btcstaking/v1/query.proto

```diff
@@ -345,3 +351,20 @@ message QueryLargestBtcReOrgResponse {
   // RollbackTo is the BTC block header which we rollback to
   babylon.btclightclient.v1.BTCHeaderInfoResponse rollback_to = 3;
 }
+
+
+// QueryParamsVersionsRequest is the request type for the
+// Query/ParamsVersions RPC method.
+message QueryParamsVersionsRequest {
+  // pagination defines an optional pagination for the request.
```

Comment:
Nit, add annotations to avoid generating code that will not be used, e.g.:

```suggestion
  option (gogoproto.equal) = false;
  option (gogoproto.goproto_getters) = false;
  
  // pagination defines an optional pagination for the request.
```